snap/snapcraft.yaml

```diff
@@ -115,36 +115,36 @@ parts:
       # shallow clone the submodules
       scripts/checkout_submodules.py --shallow --platform linux
 
-  zap:
-    plugin: nil
-    build-environment:
-      - ZAP_VERSION: v2024.03.16-nightly
-    build-packages:
-      - wget
-      - unzip
-    override-build: |
-
-      if [[ $SNAP_ARCH == "arm64" ]]; then
-        # Download and unzip the prebuilt ZAP (Zigbee Cluster Library configuration tool and generator) binary for the ARM64 architecture
-        wget --no-verbose https://github.com/project-chip/zap/releases/download/$ZAP_VERSION/zap-linux-$SNAP_ARCH.zip
-        unzip -o zap-linux-$SNAP_ARCH.zip
-
-        # Define the environment needed for the app build
-        echo "export ZAP_INSTALL_PATH=$PWD" >> env
-      fi
+  # zap:
+  #   plugin: nil
+  #   build-environment:
+  #     - ZAP_VERSION: v2024.03.16-nightly
+  #   build-packages:
+  #     - wget
+  #     - unzip
+  #   override-build: |
+
+  #     if [[ $SNAP_ARCH == "arm64" ]]; then
+  #       # Download and unzip the prebuilt ZAP (Zigbee Cluster Library configuration tool and generator) binary for the ARM64 architecture
+  #       wget --no-verbose https://github.com/project-chip/zap/releases/download/$ZAP_VERSION/zap-linux-$SNAP_ARCH.zip
+  #       unzip -o zap-linux-$SNAP_ARCH.zip
+
+  #       # Define the environment needed for the app build
+  #       echo "export ZAP_INSTALL_PATH=$PWD" >> env
+  #     fi
 
   lighting:
-    after: [libgpiod, test-blink, connectedhomeip, zap]
+    after: [libgpiod, test-blink, connectedhomeip]
     plugin: nil
-    source: app
-    build-environment:
-      - ZAP_ENV: ../../zap/build/env
+    source: .
```

Comment:
Why is the `source` path changed? Setting the source to the whole project directory will result in invalidated part's cache even if unrelated code, such as tests change.

Reply:
Because I'm using the whole `app` directory to be copied to inside the `examples` directory on chip SDK.